src/test/java/com/google/research/bleth/simulator/GlobalResolverTest.java

```diff
@@ -0,0 +1,254 @@
+package com.google.research.bleth.simulator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.runner.RunWith;
+import org.junit.Test;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GlobalResolverTest {
+    private static final Location ZERO_ON_ZERO_COORDINATE = new Location(0, 0);
+    private static final Location ONE_ON_ONE_COORDINATE = new Location(1, 1);
+    private static final Location ZERO_ON_ONE_COORDINATE = new Location(0, 1);
+    private static final Location ONE_ON_ZERO_COORDINATE = new Location(1, 0);
+    private static final Location TWO_ON_TWO_COORDINATE = new Location(2, 2);
+    private static final BeaconFactory BEACON_FACTORY = new BeaconFactory();
+
+    @Test
+    public void estimatedBoardInitializedEmpty() {
+        RealBoard realboard = new RealBoard(2, 2);
+        Beacon beacon = createRandomBeaconOnLocation(ZERO_ON_ZERO_COORDINATE, realboard);
+
+        GlobalResolver resolver = GlobalResolver.createResolver(/* rowsNum= */ 2,
+                                                                /* colsNum= */ 2,
+                                                                ImmutableList.of(beacon));
+
+        assertThat(isBoardEmpty(resolver.getBoard(), 2, 2)).isTrue();
+    }
+
+    @Test
+    public void firstEstimateWithoutReceiveInformation_EstimatedBoardIsEmpty() {
+        RealBoard realboard = new RealBoard(2, 2);
+        Beacon beacon = createRandomBeaconOnLocation(ZERO_ON_ZERO_COORDINATE, realboard);
+        GlobalResolver resolver = GlobalResolver.createResolver(/* rowsNum= */ 2,
+                                                                /* colsNum= */ 2,
+                                                                ImmutableList.of(beacon));
+
+        resolver.estimate();
+
+        assertThat(isBoardEmpty(resolver.getBoard(), 2, 2));
+    }
+
+    @Test
+    public void firstEstimateAfterReceivingZeroTransmissions_EstimatedBoardIsEmpty() {
```

Comment:
Nit: the "outcome" part of the name (after the underscore) should start again with a lowercase letter.
(Applies to other tests as well)